samples/DispatcherSample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using DispatcherSample.Models;
+
+namespace DispatcherSample.Controllers
+{
+    public class HomeController : Controller
```

Comment:
We can't use MVC from this repo, the sample should just be a `Startup.cs` and `Program.cs` for now